tests/dex_test.go

```diff
@@ -98,3 +98,22 @@ func TestPlaceOrders(t *testing.T) {
 		testCase.run(t, app)
 	}
 }
+
+func TestPlaceImposterOrders(t *testing.T) {
+	app := processblock.NewTestApp()
+	p := processblock.DexPreset(app, 3, 2)
+	p.DoRegisterMarkets(app)
+
+	for _, testCase := range []TestCase{
```

Comment:
Can we add a unit test for sending multiple bank transactions in a same block to test out the defer behavior?

Reply:
+1 ideally execute them in parallel on a shared cache if possible